### PR TITLE
#167190434 logs out a user

### PR DESCRIPTION
**Pull request by Lundii**
####  What does this PR do?
It creates a route for logging out a user

#### Description of Task to be completed?
- add a route for logging out.
- created a blacklist table for places logged out tokens yet to expire
- create a cron job using node-cron to schedule a time for deleting expired tokens in the blacklist

#### How should this be manually tested?
- fetch or pull this branch
- run npm install to get the packages used
- login as a registered user and save the token received
- logout as a user using `POST api/v1/users/logout`
- the token received during login is no longer valid.  You can test it by logging out again using the 
   token

#### Any background context you want to provide?
- when a user logs out, the token should be invalid if they try to use it again on a protected route, 
   even if the token is yet to expire.

#### What are the relevant pivotal tracker stories?
[#167190434](https://www.pivotaltracker.com/story/show/167190434)

#### Screenshots (if appropriate)
No

#### Questions:
No

### CODE REVIEW

**Review by encodedBicoding — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -66,4 +68,4 @@
       "git add"
     ]
   }
-}
+}
```
Comment (topseySuave) [inline package.json]: Add a new line for the page

File: server/helpers/blacklistToken.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  /* istanbul ignore next */
+  /**
+   * @static
+   * @returns {null} - null
+   * @memberof Blacklist
+   */
+  static async startCronJob() {
+    cron.schedule('59 23 * * *', () => {
```
Comment (topseySuave) [inline server/helpers/blacklistToken.js:49]: Can you add a comment explaining what this does.

Comment (Lundii) [reply server/helpers/blacklistToken.js:49]: Okay, done.

**Review by cvjude — APPROVED** [review]

**Review by Hadeneekeh — APPROVED** [review]
LGTM! Great work Monday.

**Review by obayomi96 — APPROVED** [review]
Good job @Lundii

**Review by topseySuave — APPROVED** [review]

**Pull request merged**